app/src/main/java/ch/epfl/sdp/peakar/general/InitActivity.java

```diff
@@ -38,12 +40,22 @@
 public class InitActivity extends AppCompatActivity {
 
     private MultiplePermissionsListener allPermissionsListener;
+    private Context mContext;
 
     @Override
     protected void onCreate(Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
+
+        mContext = this;
+
```

Comment:
mContext is not used and can be removed.
```suggestion

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
```